src/components/Alarm/Alarm.tsx

```diff
@@ -87,13 +90,8 @@ const Alarm: FC<Props> = (props: Props) => {
   };
 
   useEffect(() => {
-    /**
-     * Alarms polling dispatch action will be fired when the component is mounted
-     */
-    props.startUpdateAlarms();
     return onUnmount;
```

Comment:
Is this a good practice?

Reply:
In react this is how we can execute a function on component unmount. We have to implement that as a return function in useEffect hook